docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc

```diff
@@ -68,6 +68,8 @@ spec:
 $ oc wait --for condition=Ready --timeout=300s Nova/nova
 ----
 
+. Complete xref:adopting-compute-services-to-the-data-plane_data-plane[Adopting Compute services to the {rhos_acro} data plane]
```

Comment:
```suggestion
. Complete the steps in xref:adopting-compute-services-to-the-data-plane_data-plane[Adopting Compute services to the {rhos_acro} data plane].
```
@bogdando Wouldn't customers have already completed "Adopting the Compute services to the RHOSO data plane? Should this step be a prerequisite instead?

Reply:
Technically, however, it is easy to miss that crucial step if surfing through the upgrade guide as a separate document. Let's keep it as a heads up?

Reply:
This is not a prerequisite, as it the configuration switch happens in the middle, not in the beginning of the Compute services adoption, and ends after EDPM adoption.

Reply:
OK, sounds good.